SU2_CFD/src/solvers/CFEASolver.cpp

```diff
@@ -1791,7 +1798,70 @@ void CFEASolver::BC_Clamped_Post(CGeometry *geometry, CNumerics *numerics, CConf
 
 }
 
-void CFEASolver::BC_DispDir(CGeometry *geometry, CNumerics *numerics, CConfig *config, unsigned short val_marker) {
+void CFEASolver::BC_Sym_Plane(CGeometry *geometry, CNumerics *numerics, const CConfig *config, unsigned short val_marker) {
+
+  if (geometry->GetnElem_Bound(val_marker) == 0) return;
+
+  const bool dynamic = config->GetTime_Domain();
+
+  /*--- Determine axis of symmetry based on the normal of the first element in the marker. ---*/
+
+  const su2double* nodeCoord[4] = {nullptr};
```

Comment:
We have N_POINTS_QUADRILATERAL (and others) in case useful throughout the FEA solver. I know it can be a quad or triangle here, but you might look elsewhere to replace things where it can be more clear

Reply:
I've changed it to MAXNNODES_2D to be consistent to what was already there for 3D elements.
Thanks for the review :+1:

Reply:
👍 You're welcome. Thanks for all of the contributions you've been making across SU2.